Description: Fix missing gettextutil import
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-01-29

Index: keystone/keystone/common/utils.py
===================================================================
--- keystone.orig/keystone/common/utils.py	2014-06-30 23:51:56.000000000 +0800
+++ keystone/keystone/common/utils.py	2014-06-30 23:51:58.000000000 +0800
@@ -35,6 +35,7 @@
 from keystone.openstack.common import log
 from keystone.openstack.common import strutils
 
+from keystone.openstack.common.gettextutils import _
 
 CONF = config.CONF
 
