Description: Fix missing gettextutil import
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-01-29

Index: keystone/keystone/common/utils.py
===================================================================
--- keystone.orig/keystone/common/utils.py	2014-08-09 09:47:34.000000000 +0800
+++ keystone/keystone/common/utils.py	2014-08-09 09:50:05.000000000 +0800
@@ -34,6 +34,7 @@
 from keystone.openstack.common import strutils
 from six import moves
 
+from keystone.openstack.common.gettextutils import _
 
 CONF = config.CONF
 
