Description: Fix missing gettextutil import
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-01-29

Index: keystone/keystone/common/utils.py
===================================================================
--- keystone.orig/keystone/common/utils.py	2014-03-09 13:19:36.000000000 +0800
+++ keystone/keystone/common/utils.py	2014-03-09 13:26:57.000000000 +0800
@@ -33,6 +33,7 @@
 from keystone.openstack.common import strutils
 from six import moves
 
+from keystone.openstack.common.gettextutils import _
 
 CONF = config.CONF
 
