Description: Fix missing gettextutil import
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-01-29

Index: keystone/keystone/common/utils.py
===================================================================
--- keystone.orig/keystone/common/utils.py	2014-12-22 12:11:32.000000000 +0800
+++ keystone/keystone/common/utils.py	2014-12-22 12:23:55.000000000 +0800
@@ -34,6 +34,7 @@
 from keystone.i18n import _, _LE, _LW
 from keystone.openstack.common import log
 
+from keystone.openstack.common.gettextutils import _
 
 CONF = config.CONF
 
