Description: Fix missing gettextutil import
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-01-29

--- keystone-2014.1~b2.orig/keystone/common/utils.py
+++ keystone-2014.1~b2/keystone/common/utils.py
@@ -33,6 +33,7 @@ from keystone import exception
 from keystone.openstack.common import log
 from six import moves
 
+from keystone.openstack.common.gettextutils import _
 
 CONF = config.CONF
 
