--- a/etc/keystone.conf
+++ b/etc/keystone.conf
@@ -78,7 +78,7 @@
 # SQLAlchemy connection string for the reference implementation registry
 # server. Any valid SQLAlchemy connection string is fine.
 # See: http://bit.ly/ideIpI
-sql_connection = sqlite:///keystone.db
+sql_connection = _DBC_URL_
 backend_entities = ['UserRoleAssociation', 'Endpoints', 'Role', 'Tenant',
                     'User', 'Credentials', 'EndpointTemplates', 'Token',
                     'Service']
